### PR TITLE
OOP School Library: - Feature/setup classes 👨🏾‍💻 💻 

### PR DESCRIPTION
**Pull request by Bennyjoez**
## OOP School Library: - Feature/setup classes 👨🏾‍💻 👨🏾‍💻 
In this milestone, I have managed to do the following: 
 - Added the Person class and provided it's methods, including the private ones. 🎏 
 - Added the student class that extends the person class and the public method. 🎏 
 - Added the Teacher class that extends the person class and its methods. 🎏 
 - Setup rubocop. 🎏 
 - Fixed rubocop errors. 🎏 
 
💯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/microvese-projects/OOP_school_library.git/issues).
```
Comment (Alfredbis29) [inline README.md]: - [ ] I wanted to kindly bring to your attention a broken link in the README file . The `Open Issue` link, unfortunately, seems to be leading to an error page. Please fix this :point_down: 
Screenshot from 2023-06-26 16-58-55

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @Bennyjoez 
 Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark:  Your Work looks Professional 
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Your Class are well structured according to the requirements 
 
 ## Required Changes 
- [x] Check under the review

 ## Optional suggestions 
N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Bennyjoez,

## Highlights!
- Good code organization and classes ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bennyjoez** [comment]
Thank you @julie-ify  @Alfredbis29

**Pull request merged**